src/styles/review.css

```diff
@@ -0,0 +1,43 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 100%;
+  height: 16px;
+}
+
+.card__review {
+  margin-top: 16px;
+}
+
+.star {
+  display: flex;
+  align-self: flex-start;
+}
+
+.card__star {
+  margin-top: 16px;
+}
+
+.star__star {
+  display: inline-block;
+  margin: 2px 6px 2px 2px;
+  height: 12px;
+  width: 12px;
+
+  background-image: url("../images/star.svg");
+  background-size: cover;
+}
+
+.star :nth-child(-n+4) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
You need to be able to set active stars with a modifier in HTML. This CSS code doesn't seem to support that.